rust/transformer/src/derivation.rs

```diff
@@ -73,6 +73,16 @@ pub struct License {
 
 #[derive(Deserialize, Clone, Debug)]
 pub struct Src {
+    #[serde(deserialize_with = "deserialize_null_default")]
     pub url: String,
```

Comment:
Can't we just do this:

```suggestion
    pub url: Option<String>,
```

And if it's None we just don't add the external reference?

Reply:
sure, the only difference is that we wouldn't throw a deserialization error with `"src" = {}`. would be fine with me.